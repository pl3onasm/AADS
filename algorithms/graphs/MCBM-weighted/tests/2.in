MAX

1 4 10
1 5 23
1 6 30
1 7 40
2 4 5
2 5 20
2 6 10
2 7 60
3 4 18
3 5 20
3 6 20
3 7 30
