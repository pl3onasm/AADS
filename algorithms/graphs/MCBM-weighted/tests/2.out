
--------------------
 Network
 Vertices: 7
 Edges: 12
--------------------
  1[4]: 4(1,10), 5(1,23), 6(1,30), 7(1,40)
  2[4]: 4(1,5), 5(1,20), 6(1,10), 7(1,60)
  3[4]: 4(1,18), 5(1,20), 6(1,20), 7(1,30)
  4[0]
  5[0]
  6[0]
  7[0]
--------------------

--------------------
 Matching
 Cardinality: 3
 Max cost: 110.00
--------------------
  3 -- 5
  2 -- 7
  1 -- 6
--------------------